app/routes/message_blueprint.py

```diff
@@ -1,7 +1,7 @@
 from flask import Blueprint
 
-from app.controllers.message_controller import messages
```

Comment:
We must find a patter to our imports

Reply:
All imports must call the files and then their functions, as per the script passed. This should be the pattern to follow. The script is correct and meets the requirement.